docs/src/pages/[platform]/connected-components/geo/react.mdx

```diff
@@ -121,3 +121,7 @@ with provided `latitude` and `longitude` properties is required.
     import(`./fragments/location-search-standalone.${platform}.mdx`)
   }
 </Fragment>
+
+<Alert variation="info" heading="Create React App setup">
```

Comment:
Can you move this up below Quick Start?

Reply:
Done! 38a043afe5fc0cad2119d139c063e4c64256f576